src/interface/src/app/map/map.component.ts

```diff
@@ -581,9 +581,13 @@ export class MapComponent implements AfterViewInit, OnDestroy, OnInit {
         return false;
       case 2:
       default:
-        // In 2 map view, only the 1st and 2nd map are shown regardless of selection
-        if (index === 0 || index === 1) {
-          // TODO: 2 map view might go away or the logic here might change
+        // In 2 map view, if the 1st or 2nd map are selected, show maps 1 and 2
+        // Otherwise, show maps 3 and 4
+        // TODO: 2 map view might go away or the logic here might change
+        if (
```

Comment:
Nit: could this be simplified to 

  return Math.floor(this.mapViewOptions$.getValue().selectedMapIndex / 2) === Math.floor(index / 2);

?